src/withSort.js

```diff
@@ -0,0 +1,35 @@
+import { faSort } from "@fortawesome/free-solid-svg-icons";
+import { useEffect } from "react"
+
+const withSort = function(Component,property) {
+    return function(props) {
+        
+        let myProp = {};
+        function customSort(objA,objB) {
+            if(objA[property] < objB[property]) {
+                return -1;
+            } 
+            else if(objA[property] > objB[property]) {
+                return 1;
+            }
+            return 0;
+        }
+
+        for(let prop in props) {
+            if(Array.isArray(props[prop])) {
```

Comment:
we shouldn't check like this, there can be other prop value which is array. Keep it generic,